### ISSUE TITLE
How to obtain shape information of a three-dimensional sparse tensor, similar to (32,32,32)?

### ISSUE THREAD

**Issue by Shixiaomeng7** [mit-han-lab/torchsparse#261]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I want to output the specific shapes of each dimension of sparse voxels, but use. spatial_ Shape and other commands always prompt 'SparseTensor' object has no attribute 'spatial_ Shape ', please help.

### Environment

```markdown
- GCC:
- NVCC:
- PyTorch:
- PyTorch CUDA:
- TorchSparse:
```

**Comment by ys-2020** [comment]
Hi. I think that should be `spatial_range` instead of `spatial_Shape`. The definition of `SparseTensor` can be found [here](https://github.com/mit-han-lab/torchsparse/blob/master/torchsparse/tensor.py).

**Comment by Shixiaomeng7** [comment]
> Hi. I think that should be `spatial_range` instead of `spatial_Shape`. The definition of `SparseTensor` can be found [here](https://github.com/mit-han-lab/torchsparse/blob/master/torchsparse/tensor.py).

Thank you so much for your answer! But I think I found the problem, I found the spatial_range you mentioned in Torchsparse2.0.0 was passed in as an attribute to SparseTensor, but I am using version 1.4.0. Upon checking, it was found that the only attributes passed in to SparseTensor are coords and feats, so it should not be possible to view the specific shape. Anyway, thank you so much for your answer and help me solve the problem.
